logstreams/src/main/java/io/zeebe/logstreams/processor/StreamProcessorController.java

```diff
@@ -246,6 +249,10 @@ private void reprocessEvent(final LoggedEvent currentEvent) {
           try {
             // don't execute side effects or write events
             zeebeDb.transaction(eventProcessor::processEvent);
+          } catch (final RecoverableException recoverableException) {
+            // recoverable
```

Comment:
I think logging the exception is useful here, since even if it succeeds on the next retry, it could indicate symptoms of intermittent failure.

Reply:
Yeah was not sure, since we then spam the log :sweat_smile: but we could do on debug or trace level